recipe/install_activate.bat

```diff
@@ -0,0 +1,10 @@
+set YEAR=2015
+set VER=14
+
+mkdir "%PREFIX%\etc\conda\activate.d"
+
+echo @echo on > "%PREFIX%\etc\conda\activate.d\vs%YEAR%_compiler_vars.bat"
+echo SET "cross_compiler_target_platform=%cross_compiler_target_platform%" >> "%PREFIX%\etc\conda\activate.d\vs%YEAR%_compiler_vars.bat"
```

Comment:
We don't care that this will override a set `cross_compiler_target_platform`, right?
I assume we don't really, since it does not make much sense to have it set to another than `X` when you use `vs2015_X`.

Reply:
> We don't care that this will override a set cross_compiler_target_platform, right?

Yes

> I assume we don't really, since it does not make much sense to have it set to another than X when you use vs2015_X.

exactly.